### PR TITLE
Feature: Promotion in payment modal

### PR DESCRIPTION
**Pull request by creativejuiz**

### CODE REVIEW

**Comment by GeekPress** [comment]
@creativejuiz I don't valide the PR yet because of this comment:
https://github.com/wp-media/imagify-plugin/pull/44/files#r92347584

File: inc/api/imagify.php [hunk 1]
```diff
@@ -134,7 +147,7 @@ class Imagify {
     /**
      * The Imagify API endpoint
      */
-    const API_ENDPOINT = 'https://app.imagify.io/api/';
+    const API_ENDPOINT = IMAGIFY_API_URL;
```
Comment (creativejuiz) [inline inc/api/imagify.php]: Good to know. Thank you @GeekPress :)

**Comment by creativejuiz** [comment]
@GeekPress @Tabrisrp updated https://github.com/wp-media/imagify-plugin/pull/44/commits/9ecf0192b3cc30fa0d1c7949f65ffaf93c23a8d5

**Comment by Tabrisrp** [comment]
Pull request should be in 1.6.3 no? And with updated version and readme

**Comment by GeekPress** [comment]
@Tabrisrp Yes you're right! poke @creativejuiz

**Comment by jbma** [comment]
@creativejuiz can you pull request in 1.6.3?

**Comment by creativejuiz** [comment]
Seems like fixed :D

**Pull request merged**